runtime/kusama/src/lib.rs

```diff
@@ -1667,6 +1674,10 @@ sp_api::impl_runtime_apis! {
 			})
 		}
 
+		fn session_ee_params_by_parent_hash(parent_hash: Hash) -> Option<ExecutorParams> {
```

Comment:
Also here - I think this shouldn't be available in Kusama initially. It implements v2 so you can remove this, if the method is tagged with v3.